app/mailers/user_mailer.rb

```diff
@@ -84,4 +84,9 @@ def letter_reminder(email)
   def letter_expired(email)
     mail(to: email, subject: t('user_mailer.letter_expired.subject'))
   end
+
+  def confirm_email_and_reverify(email, account_recovery_request)
+    @token = account_recovery_request&.request_token
```

Comment:
Why do we need to guard against a nil account recovery request? It seems like if we are trying to send an email with a nil account recovery request then we should raise, right?